packages/core/src/utilities/background-color/background-color.example.html

```diff
@@ -7,12 +7,14 @@
 'secondary-lightest'], ['error', 'error-dark', 'error-darkest', 'error-light', 'error-lighter',
 'error-lightest'], ['base', 'white', 'background', 'background-inverse', 'transparent'] ]; -%> <%
```

Comment:
Maybe we should add `black` here too

Reply:
Add black where exactly? https://user-images.githubusercontent.com/1449852/70641410-93e99c00-1c0b-11ea-9003-5973e43400a2.png

Reply:
Oh i see you reorganized it, Originally I was looking at putting it with these colors:
`['base', 'white', 'background', 'background-inverse', 'transparent']`